README.md

```diff
@@ -11,3 +11,9 @@ pip install -e .
 ## Usage
 
 This package is mainly intended for working with clinical trials data dump in xml files.
+
+## Data
+
+To download data for your analysis, follow the description from [here](https://clinicaltrials.gov/ct2/resources/download#DownloadAllData).
+
+Description of the ClinicalTrials schema: https://prsinfo.clinicaltrials.gov/ProtocolRecordSchema.xsd
```

Comment:
Codacy has a fix for the issue: [Bare URL used](https://app.codacy.com/gh/WojciechKusa/clinical-trials/pullRequest?prid=9912303)
```suggestion
Description of the ClinicalTrials schema: <https://prsinfo.clinicaltrials.gov/ProtocolRecordSchema.xsd>
```